Add Jogger (#260)
myfreeweb/awesome-gtk: README.md

## Health and Fitness

- [BMI](https://github.com/PhilippKosarev/bmi) - Body mass index (BMI) calculator `#python` `#gtk4` `#libadwaita`.
- [Dosage](https://github.com/diegopvlk/Dosage) - Medication tracker `#gjs` `#gtk4` `#libadwaita`.
- [Health](https://apps.gnome.org/Health) - Fitness goals tracker `#rust` `#gtk4` `#libadwaita` `#gnome`.

> - [Jogger](https://codeberg.org/baarkerlounger/jogger) - App for Gnome Mobile to Track running and other workouts `#rust` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Jogger](https://codeberg.org/baarkerlounger/jogger) - Running (and other workout) tracker for GNOME Mobile with import from Fitotrack  `#rust` `#gtk4` `#libadwaita`.
```
